modules/platform/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/management/BinariesObject.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.ecm.restapi.server.jaxrs.management;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.Response.Status.CONFLICT;
+
+import javax.ws.rs.DELETE;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import org.nuxeo.ecm.core.blob.DocumentBlobManager;
+import org.nuxeo.ecm.core.blob.binary.BinaryManagerStatus;
+import org.nuxeo.ecm.webengine.model.WebObject;
+import org.nuxeo.ecm.webengine.model.impl.AbstractResource;
+import org.nuxeo.ecm.webengine.model.impl.ResourceTypeImpl;
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * Endpoint to manage the binaries.
+ *
+ * @since 11.3
+ */
+@WebObject(type = ManagementObject.MANAGEMENT_OBJECT_PREFIX + "binaries")
+@Produces(APPLICATION_JSON)
+public class BinariesObject extends AbstractResource<ResourceTypeImpl> {
+
+    /**
+     * Garbage collect the unused (orphaned) binaries.
+     * 
+     * @return {@link BinaryManagerStatus} if no gc is in progress, otherwise a
+     *         {@link javax.ws.rs.core.Response.Status#CONFLICT}
+     */
+    @DELETE
+    @Path("orphaned")
+    public Response garbageCollectBinaries() {
+        DocumentBlobManager documentBlobManager = Framework.getService(DocumentBlobManager.class);
+
+        if (!documentBlobManager.isBinariesGarbageCollectionInProgress()) {
+            BinaryManagerStatus binaryManagerStatus = documentBlobManager.garbageCollectBinaries(true);
+            return Response.ok(binaryManagerStatus).build();
+        }
+
+        return Response.status(CONFLICT).build();
+    }
```

Comment:
What about throwing an exception instead of returning a `Response`? We have this pattern in the `BulkObject`.

Furthermore, we recently saw an issue when using the `Response` pattern in [JSONDocumentObject](https://github.com/nuxeo/nuxeo/blob/2387c0ba417810c3b68fd8d51795b46cdee6f579/modules/platform/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/JSONDocumentObject.java#L85). The  `Response` has `application/json` as content type whereas the entity is a simple String.